src/components/farm-info-card/FarmOverview.tsx

```diff
@@ -0,0 +1,195 @@
+    name: poolData.name,
+    reserve: poolData.reserve
+      ? formatBNToShortString(poolData.reserve, 18)
+      : "-",
+    apy: poolData.apy
+      ? `${poolData.apy.toLocaleString(undefined, {
+        maximumFractionDigits: 2,
+      })}%`
+      : "-",
+    volume: poolData.volume ? `$${poolData.volume}` : "-",
+    userBalanceUSD: formatBNToShortString(
+      userShareData?.usdBalance || Zero,
+      18,
+    ),
+    tokens: poolData.tokens.map((coin) => {
+      const token = TOKENS_MAP[coin.symbol]
+      return {
+        symbol: token.symbol,
+        name: token.name,
+        icon: token.icon,
+        value: formatBNToString(coin.value, token.decimals, formattedDecimals),
+      }
+    }),
+  }
+  const hasShare = !!userShareData?.usdBalance.gt("0")
+
+  const masterchefContract = new ethers.Contract(
+    AXIAL_MASTERCHEF_CONTRACT_ADDRESS[43114],
+    masterchef,
+    library,
+  )
+
+  async function onClickWithdraw() {
+    console.log("withdraw")
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    await masterchefContract.withdrawAll()
+  }
+
+  async function onClickDeposit() {
+    console.log("deposit")
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    await masterchefContract.depositAll()
+  }
+
+  async function onClickClaim() {
+    console.log("claim")
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    await masterchefContract.claim()
+  }
+  console.log({ tokens: formattedData.tokens })
+  return (
+    <div
+      className={classNames("poolOverview", {
+        outdated: isOutdated || shouldMigrate,
+      })}
+    >
+      <div className="left">
+        <div className="titleAndTag">
+          <h4 className="title">{formattedData.name}</h4>
+          {(shouldMigrate || isOutdated) && <Tag kind="warning">OUTDATED</Tag>}
+          {poolData.isPaused && <Tag kind="error">PAUSED</Tag>}
+        </div>
+        {hasShare && (
+          <div className="balance">
+            <span>{t("balance")}: </span>
+            <span>{`$${formattedData.userBalanceUSD}`}</span>
+          </div>
+        )}
+        <div className="tokens">
+          <span style={{ marginRight: "8px" }}>[</span>
+          {formattedData.tokens.map(({ symbol, icon }) => (
+            <div className="token" key={symbol}>
+              <img alt="icon" src={icon} />
+              <span>{symbol}</span>
+            </div>
+          ))}
+          <span style={{ marginLeft: "-8px" }}>]</span>
+        </div>
+      </div>
+
+      <div className="right">
+        <div className="poolInfo">
+          {formattedData.apy && (
+            <div className="margin">
+              <span className="label">{`${t("apy")}`}</span>
+              <span>{formattedData.apy}</span>
+            </div>
+          )}
+          <div className="margin">
+            <span className="label">TVL</span>
+            <span>{`$${formattedData.reserve}`}</span>
+          </div>
```

Comment:
Avoid pushing commented out code, if you do because you have a reason, you need to add a description for why is commented out.